ui/app/templates/vault/cluster.hbs

```diff
@@ -12,12 +12,21 @@
       {{#if flash.componentName}}
         {{component flash.componentName content=flash.content}}
       {{else}}
-        <AlertPopup
-          @type={{message-types flash.type}}
-          @message={{flash.message}}
-          @close={{close}}
-          @isPreformatted={{flash.preformatted}}
-        />
+        {{#let (hash info="highlight" success="success" danger="critical" warning="warning") as |color|}}
```

Comment:
I opted to do this in the hbs template instead of making an external helper since I didn't see a need for these mappings to be used elsewhere once we transition completely to using `HDS` components. For example, `AlertInline` should eventually have `@type="danger"` updated to `@type="critical"` to match HDS language. 
But would love some feedback here!

Reply:
I think it makes sense to do this in the template for now :)